raiden_installer/web.py

```diff
@@ -329,7 +337,7 @@ def _run_track_transaction(self, **kw):
         configuration_file_name = kw.get("configuration_file_name")
         tx_hash = kw.get("tx_hash")
         time_elapsed = 0
-
+        time.sleep(30)
```

Comment:
is this sleep intentional?

Reply:
how did that slip through?

Reply:
We have a problem though that track transaction is called before the node has received the tx_hash. There is already a 30s sleep in place